### DISCUSSION TITLE
How to pass array of path to resolvers to buildSchema

### DISCUSSION THREAD

**Discussion by ludersGabriel** [MichalLytek/type-graphql#1469] [Q&A] [answered]
Hi! I used to do this:

![image](https://github.com/MichalLytek/type-graphql/assets/57847215/03ab3d14-f172-427c-97cd-c12b7c1d2552)

But now it is giving me "string is not assignable to type function" and when I execute, it gives:

![image](https://github.com/MichalLytek/type-graphql/assets/57847215/4de7c352-4147-42ac-819c-ac99614fbfdb)

Did it change how its done with the 2.0?

**Comment by carlocorradini** [comment, marked as answer]
Glob paths are removed from V2.
You need to pass an array of (classes) resolvers:
```ts
// ...
resolvers: [Resolver1, Resolver2, ...]
```

**Reply by ludersGabriel** [reply]
I see. Why was it removed? It is kinda bad to pass everything by hand everytime I create a new resolver

**Reply by carlocorradini** [reply]
See https://github.com/MichalLytek/type-graphql/issues/1418#issuecomment-1619829172
